src/bin/casc.rs

```diff
@@ -35,3 +36,32 @@ fn main() -> std::io::Result<()> {
         Ok(s) => out_file.write_all(s.as_bytes()),
     }
 }
+
+// Create a list of policy files
+// Display info and proceed on errors.  As long as we have at least one file, errors should be
+// considered non-fatal
+fn get_policy_files(filenames: Vec<String>) -> Vec<String> {
+    let mut policy_files = Vec::new();
+    for file in filenames {
+        for entry in WalkDir::new(file) {
+            match entry {
+                Ok(entry) => {
+                    if entry.file_type().is_file()
+                        && entry.path().extension().unwrap_or_default() == "cas"
+                    {
+                        let filename = entry.path().display().to_string();
+                        policy_files.push(filename);
+                    }
+                }
+                Err(e) => {
+                    if let Some(path) = e.path() {
+                        println!("{}: {}", path.display(), e);
```

Comment:
Good point.  I'll switch to eprintln!.  I think this might actually be the only place this is missing though.  Explicit printing should only happen in this one short file.